lib/miq_automation_engine/service_models/miq_ae_service_tenant.rb

```diff
@@ -1,17 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceTenant < MiqAeServiceModelBase
-    expose :id
-    expose :domain
-    expose :subdomain
-    expose :name
```

Comment:
1. Think we want to keep `name`, `description`, `ancestors`, `siblings`, `parents`, `children`